examples/kitchensink/config/admin.js

```diff
@@ -6,4 +6,9 @@ module.exports = ({ env }) => ({
   apiToken: {
     salt: env('API_TOKEN_SALT', 'example-salt'),
   },
+  transfer: {
+    token: {
```

Comment:
We're also using apiToken just above, I thought it would be better to keep some consistency, wdyt?

Reply:
I feel more like apiToken is referring to the feature itself (or should have been plural) since most other things are. But it's quite a minor issue, I don't think we have any clear pattern at the moment.